domains/runtime/evm/src/lib.rs

```diff
@@ -1330,6 +1358,23 @@ impl_runtime_apis! {
             }
         }
 
+        fn find_first_inherent_extrinsic(extrinsics: &Vec<<Block as BlockT>::Extrinsic>) -> Option<u32> {
+            for (index, extrinsic) in extrinsics.iter().enumerate() {
+                let is_inherent_extrinsic = match &extrinsic.0.function {
+                    RuntimeCall::Timestamp(call) => Timestamp::is_inherent(call),
+                    RuntimeCall::ExecutivePallet(call) => ExecutivePallet::is_inherent(call),
+                    RuntimeCall::Messenger(call) => Messenger::is_inherent(call),
+                    RuntimeCall::Sudo(call) => Sudo::is_inherent(call),
+                    RuntimeCall::EVMNoncetracker(call) => EVMNoncetracker::is_inherent(call),
+                    _ => false,
+                };
```

Comment:
Is there a test that makes sure this list is kept up to date?
(We've made the mistake of adding pallets and not updating pallet lists before.)

Reply:
Found a way to avoid this.

Reply:
Thank you for this fix!